src/Castle.Core.Tests/Castle.Core.Tests-VS2017.csproj

```diff
@@ -1,22 +1,45 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+	
 	<Import Project="..\..\buildscripts\common.props"></Import>
+
+	<PropertyGroup>
+		<TargetFrameworks Condition="'$(IsOSX)'=='true'">netcoreapp1.1</TargetFrameworks>
+		<TargetFrameworks Condition="'$(IsLinux)'=='true'">netcoreapp1.1</TargetFrameworks>
+		<TargetFrameworks Condition="'$(IsWindows)'=='true'">net461;netcoreapp1.1</TargetFrameworks>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<RuntimeIdentifier Condition="'$(IsOSX)'=='true'AND'$(TargetFramework)'=='net461'">osx.10.12-x64</RuntimeIdentifier>
+		<RuntimeIdentifier Condition="'$(IsOSX)'=='true'AND'$(TargetFramework)'=='netcoreapp1.1'"></RuntimeIdentifier>
+		<RuntimeIdentifier Condition="'$(IsLinux)'=='true'AND'$(TargetFramework)'=='net461'">ubuntu.16.04-x64</RuntimeIdentifier>
+		<RuntimeIdentifier Condition="'$(IsLinux)'=='true'AND'$(TargetFramework)'=='netcoreapp1.1'"></RuntimeIdentifier>
+		<RuntimeIdentifier Condition="'$(IsWindows)'=='true'">win7-x64</RuntimeIdentifier>
```

Comment:
Good spot. This is a real source of trouble. I thought I could easily vary the <OutputType>Exe</OutputType> when converting the tests to a console app using NUnitLite. Turns out that without the RuntimeIdentifier being set the sdk does not honour this setting. This is me mucking around to get it working on both windows and linux!